setup/setup.go

```diff
@@ -197,6 +198,16 @@ func getProvidersFromConfig(credentials Config, include string, log logr.Logger)
 	return providers, nil
 }
 
+func includeProvider(include string, provider string) bool {
+	if provider == simpleprovider.ProviderName {
+		return false
```

Comment:
The simple type does not allow access to the idp and therefore can not create a new credential. It needs to be set up manually.
When updating credentials however, the old one will be kept.